### PR TITLE
Added delete functionality

### PR DESCRIPTION
**Pull request by Lillian-wang**

### CODE REVIEW

File: housing_django/static/js/app.js [hunk 1]
```diff
@@ -15,14 +15,19 @@ $(document).ready(function() {
     el: $('.form-view'),
     collection: applicantsCollection,
   });
+  var applicantDeleteForm = new ApplicantDeleteForm({
+    el:$('.delete-view'),
```
Comment (comolongo) [inline housing_django/static/js/app.js]: space after ':'

File: housing_django/static/js/app.js [hunk 2]
```diff
@@ -15,14 +15,19 @@ $(document).ready(function() {
...
+    collection: applicantsCollection,
+  })
   var controlView = new ControlView({
     el: $('.control-view'),
     collection: applicantsCollection,
-    applicantForm: applicantForm
+    applicantForm: applicantForm,
```
Comment (comolongo) [inline housing_django/static/js/app.js]: Remove last comma. This breaks older versions of IE.

File: housing_django/static/js/applicantdelete.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    if (applicantId) { 
+      applicant = this.collection.get(applicantId).toJSON();
+    }
+    var templateResults = this.template({
+      applicant: applicant
+    })    
+    
+    this.$el.html(templateResults);
+  },
+})
```
Comment (comolongo) [inline housing_django/static/js/applicantdelete.js]: add ;

**Review by comolongo — CHANGES_REQUESTED** [review]

File: housing_django/static/js/app.js (hunk 1 above)
Comment (Lillian-wang) [reply housing_django/static/js/app.js]: done

File: housing_django/static/js/app.js (hunk 2 above)
Comment (Lillian-wang) [reply housing_django/static/js/app.js]: done

**Review by comolongo — APPROVED** [review]

**Pull request merged**